dashboard/app/views/levels/editors/fields/_bubble_choice_sublevel.html.haml

```diff
@@ -1,18 +1,18 @@
 %legend.control-legend{data: {toggle: "collapse", target: "#bubble_choice"}}
-  Bubble Choice
+  Bubble Choice & Lesson Extras
 
 #bubble_choice.in.collapse
   - unless @level.is_a?(DSLDefined) || @level.is_a?(Unplugged)
     .field
       = f.label :display_name
-      %p Set the name that will show for this level on the parent bubble choice level. Also will override how this level is displayed on the Course Overview page.
-      = f.text_field :display_name, placeholder: '(defaults to Name)'
+      %p Set the name that will show for this level on the parent level. Also will override how this level is displayed on the Course Overview page. Limit to 25 characters
+      = f.text_field :display_name, placeholder: '(defaults to Name)', maxlength: 25, style: 'width: 50%'
```

Comment:
While this ever be translated? I'm just wondering what happens if something 25 characters long in English is longer in a different language. Same for description below.

Reply:
We don't currently support editing in any other language and we are a good ways off from that. 

I talked to Amanda about this and she said at the place she worked before they used a general rule that translations were generally max 20% longer than the english version so she suggested that I find out the max length of english characters that could fit in the space and then multiple it by 0.8 to find the max length we would enforce in order to leave extra space for translation.

Reply:
That's a nifty rule of thumb that might come in handy other places! Thanks for following up.